packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -265,12 +265,27 @@ location ~ ^/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.+)$ {
     more_clear_input_headers Accept-Encoding;
     include includes/proxy-headers.conf;
     # Non-streaming endpoints don't require HTTP/1.1 but will work as
-    # expected with it enabled. Streaming endpoints require keepalive
+    # expected with it enabled. Streaming endpoints require chunked encoding
     # functionality added in HTTP/1.1. As such, we enable HTTP/1.1 here
     # while maintaining backwards compatibility.
     include includes/http-11.conf;
     # Disable buffering to support streaming endpoints
     include includes/disable-request-response-buffering.conf;
+
+    # For the reasons stated in
+    # https://jira.mesosphere.com/browse/DCOS-42098
+    # we want to make Master Admin Router emit a 'Connection: close' header value.
+    # We use 'keepalive_timeout 0;' technique to do that.
+    # See https://forum.nginx.org/read.php?2,251560,251570
+
+    # By setting 'Connection: close' in response to the request to the Agent
+    # Admin Router the Master Admin Router indicates to the client that it should
```

Comment:
There is no Agent Admin Router involved here :)

Reply:
Let's remove the entire part "to the request to the Agent Admin Router the Master Admin Router indicates".

Reply:
What do we think about the following?

```
# Setting the 'Connection: close' header value in the HTTP response
# instructs the client to cease sending requests on the same TCP
# connection, and to close the connection after reading the response
# (see https://tools.ietf.org/html/rfc7230#section-6.6).
#
# When the response is a streaming response (a response that does
# not have a defined end) setting this header value is important for
# signaling downstream HTTP proxies that they have to close the 
# TCP connection when their own client went away).
```

Reply:
Changed accordingly